### PR TITLE
Create Rule22-30.md

### LINKED ISSUE #310: Create Rule22-30.md
Condenser pump power is 19W/gpm

### PR DESCRIPTION
**Pull request by xingcx**
resolves #310

### CODE REVIEW

File: docs/section22/Rule22-30.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** Section G3.1.3.11 Heat Rejection (Systems 7, 8, 11, 12, and 13)  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 12, 13, 7b, 8b, 12b, 13b.
+2. B-RMR is not modeled with any system that is Type-11.1, 11.2 or 11b.
+3. Primary and secondary CHW loops are modeled correctly in B-RMR.
```
Comment (mkarpman) [inline docs/section22/Rule22-30.md:16]: This RDS is about condenser loop pump power, so why do we need to check primarly/secondary CHW loop?

File: docs/section22/Rule22-30.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - Check if B-RMR is modeled with any air-side system that is Type-11.1, 11.2 or 11b, rule is not applicable to B-RMR: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-11.1", "SYS-11.2", "SYS-11B"]): RULE_NOT_APPLICABLE`
+
+    - Else, continue to the next applicability check: `else: NEXT_APPLICABILITY_CHECK`
+
+- **Check 3:** Get primary and secondary loops for B-RMR: `primary_secondary_loop_dictionary = get_primary_secondary_loops(B_RMR, "COOLING")`
+
+  - Check if primary and secondary loops are modeled correctly, continue to rule logic: `if primary_secondary_loop_dictionary["PRIMARY"] AND primary_secondary_loop_dictionary["SECONDARY"]: CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
```
Comment (mkarpman) [inline docs/section22/Rule22-30.md:39]: I think Check 3 should be removed per above.

File: docs/section22/Rule22-30.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+## Rule Logic:
+
+- For each pump in B_RMR: `for pump_b in B_RMR.RulesetModelInstance.pumps:`
+
```
Comment (mkarpman) [inline docs/section22/Rule22-30.md:43]: Why can't we just identify all loops that have FluidLoop.type="condenser" and verify that for that loop fluidLoop.pump_pwer_per_flow_rate is equivalent to 19 W/GPM?

File: docs/section22/Rule22-30.md (hunk 1 above)
Comment (xingcx) [reply docs/section22/Rule22-30.md:16]: Primary/secondary loop check is now removed.

File: docs/section22/Rule22-30.md (hunk 2 above)
Comment (xingcx) [reply docs/section22/Rule22-30.md:39]: Removed.

File: docs/section22/Rule22-30.md (hunk 3 above)
Comment (xingcx) [reply docs/section22/Rule22-30.md:43]: Modified.

**Review by mkarpman — APPROVED** [review]

**Review by supriyagoel — APPROVED** [review]

**Pull request merged**